src/utils/products.expiration.ts

```diff
@@ -31,20 +30,14 @@ export const checkProductExpiration = async () => {
 			"findOne",
 			condition_one,
 		);
-		const options = {
-			to: user.email,
-			subject: `Your products on Hackers have been expired.`,
-			html: update_product_email_template(user, sellerProducts),
-		};
-		sendEmail(options);
 
 		for (const product of sellerProducts) {
 			const condition = { where: { id: product.id } };
-			const productStatus = "Unavailable";
+			const isAvailable = false;
 			await insert_function<ProductAttributes>(
 				"Product",
 				"update",
-				{ productStatus },
+				{ isAvailable },
```

Comment:
replace this with "{ isAvailable }," with "{ !isAvailable }", after deleting the above container variable